x-pack/plugins/security_solution/public/detections/pages/detection_engine/rules/all/use_columns.tsx

```diff
@@ -68,7 +68,7 @@ const useEnabledColumn = ({ hasPermissions }: ColumnsProps): TableColumn => {
       render: (_, rule: Rule) => (
         <EuiToolTip
           position="top"
-          content={getToolTipContent(rule, hasMlPermissions, hasActionsPrivileges)}
+          content={getToolTipContent(rule, hasMlPermissions, hasActionsPrivileges, true)}
```

Comment:
Could we pass an actual `canUserCRUD` value here instead of the hardcoded `true`?

Reply:
sure